src/droid/download_manager.py

```
@@ -65,6 +66,15 @@ def __init__(self, settings: Dict[str, Any], download_queue: DownloadQueueState)
 
         self._init_android()
 
+        # Reap statuses orphaned by a killed session so the UI doesn't show
+        # phantom in-progress items from before this launch.
+        try:
+            reaped = reap_stale_statuses(self.work_dir)
+            if reaped:
+                log_error(f"Reaped {reaped} stale extraction status(es) on startup")
+        except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)